### PR TITLE
Set timeout for requests made by the /status endpoint

### PR DESCRIPTION
**Pull request by dcho5**
CLOUDBLD-3702

Signed-off-by: Daniel Cho <dacho@redhat.com>

### CODE REVIEW

File: cachito/web/status.py [hunk 1]
```diff
@@ -25,7 +26,7 @@
     "*": {DATABASE, RABBITMQ},
 }
 
-no_retry_session = requests.Session()
+retry_session = get_requests_session()
```
Comment (chmeliik) [inline cachito/web/status.py:29]: We should keep using the `no_retry_session`. The purpose of the status endpoint is to check the status at a given moment in time, retrying on failures is not desirable.

File: cachito/web/status.py [hunk 2]
```diff
@@ -37,8 +38,9 @@ def _service_ok(url):
     :param str url: url to fetch
     :return: tuple (ok: bool, reason: str or None)
     """
+    config = get_worker_config()
     try:
-        resp = no_retry_session.get(url)
+        resp = retry_session.get(url, timeout=config.cachito_api_timeout)
```
Comment (chmeliik) [inline cachito/web/status.py:43]: The `cachito_api_timeout` is too long for this purpose. I would generally expect the status endpoint to respond in less than 10 seconds, for the individual requests around 3 seconds (or slightly larger than 3 seconds as recommended here https://docs.python-requests.org/en/master/user/advanced/#timeouts) should be enough.

I don't think this needs to be configurable, IMO a constant in status.py would do. But if you do want add a config option for this I won't stop you :-)

File: cachito/web/status.py [hunk 3]
```diff
@@ -26,6 +26,7 @@
 }
 
 no_retry_session = requests.Session()
+service_timeout = 5
```
Comment (chmeliik) [inline cachito/web/status.py:29]: Constants should generally be uppercase, there are some exceptions such as request sessions and logger objects but this should be uppercase IMO

File: cachito/web/status.py (hunk 2 above)
Comment (akhmanova) [reply cachito/web/status.py]: @chmeliik Do you know why api timeout is located in the worker config file?

Comment (chmeliik) [reply cachito/web/status.py]: @akhmanova it is the timeout that workers use when communicating with the API

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Pull request merged**